bigbluebutton-html5/imports/api/meetings/server/modifiers/addMeeting.js

```diff
@@ -83,7 +83,12 @@ export default function addMeeting(meeting) {
     setBy: 'temp',
   };
 
-  newMeeting.welcomeProp.welcomeMsg = newMeeting.welcomeProp.welcomeMsg.replace('event:', '');
+  newMeeting.welcomeProp.welcomeMsg = newMeeting.welcomeProp.welcomeMsg.replace(
+    'href="event:',
+    'href="',
+  );
+
+  newMeeting.welcomeProp.welcomeMsg = newMeeting.welcomeProp.welcomeMsg.replace(/href/g, 'target="_blank" href');
```

Comment:
This is going to create a duplication of the `target` prop for the second link in the `welcomeMsg` string, consider this alternative:

```
  const insertBlankTarget = (s,i) => s.substr(0, i) + ' target="_blank"' + s.substr(i);
  const linkWithoutTarget = new RegExp(`<a href="(.*?)">`, 'g');
  linkWithoutTarget.test(newMeeting.welcomeProp.welcomeMsg);

  if (linkWithoutTarget.lastIndex > 0) {
    newMeeting.welcomeProp.welcomeMsg = insertBlankTarget(
      newMeeting.welcomeProp.welcomeMsg,
      linkWithoutTarget.lastIndex - 1
    );
  }
```